lib/tasks/election.rake

```diff
@@ -59,4 +59,25 @@ namespace :election do
       puts e.message
     end
   end
+
+  desc "
+  Mark all documents of a given organisation as political
+  Usage:
+    rake election:mark_documents_as_political_for[organisation_slug]
+  "
+  task :mark_documents_as_political_for, [:slug] => :environment do |_t, args|
```

Comment:
@johnpauldickie has confirmed that we need to apply this only to documents published on GOV.UK after a certain date for each organisation, so we'll need to add a date parameter to the command